backend/src/lib/transformer/index.js

```diff
@@ -35,6 +35,7 @@ export default function (key, data, mopidy) {
       return data.new_state
     case Mopidy.GET_CURRENT_TRACK:
       const trackInfo = TransformTrack(data)
```

Comment:
Could `if (!data) return null` go before `const trackInfo = TransformTrack(data)`?

Reply:
yes 👍

Reply:
Actually no. It needs to return `{ track: null }` at the mo (which is what `TransformTrack` returns if there is no data) so the client gets a message it knows how to handle based on the message `key`. I think ultimately this could be fixed, but it's a bigger change than just adding return.